### ISSUE TITLE
2.4.7 Focus Visible (What's Next to be Loaded?)

### ISSUE THREAD

**Issue by AlanaBayless** [UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan#117]
**Were violation(s) found?**
(If Yes, describe in the comments.)
- [ ] No violations found
- [ ] Yes, violations found and all fixed
- [ ] Yes, violations found and some fixed while others are unresolved _(Add **6.Unresolved 508** label)_
- [ ] Yes, violations found but remain unresolved _(Add **6.Unresolved 508** label)_
- [ ] Flag as a potential problem _(Add **4.Help** label)_

---
### 2.4.7 Focus Visible - Level AA

Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.

[Understanding 2.4.7](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)
[How to Meet 2.4.7](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=131%2C133%2C141%2C143%2C211%2C212%2C221%2C231%2C242%2C245%2C246%2C247#focus-visible)
[How to Test 2.4.7](https://usfs.box.com/s/knn5w8jul8g0xicrnpa4cxzk07t8rc1d)

**Trusted Tester ID:**
- [x] Test 4.D focus-visible

**Accessibility Principle:** Operable (Navigable)

**Comment by AlanaBayless** [comment]
PASS

**Comment by AlanaBayless** [comment]
A visible indication of focus is provided when focus is on the interface component. Therefore the page is QA Certified.
